packages/perseus-core/src/widgets/sorter/sorter-util.ts

```diff
@@ -17,13 +20,23 @@ type SorterPublicWidgetOptions = {
 function getSorterPublicWidgetOptions(
     options: PerseusSorterWidgetOptions,
 ): SorterPublicWidgetOptions {
+    const correctToString = options.correct.join("");
```

Comment:
Minor: if we really want to ensure that different `correct` values result in different seeds, we should use `JSON.stringify()` instead of `join` here. The problem with `join` is that a value like `["1", "23"]` will produce the same seed as `["12", "3"]`.

This is unlikely to matter much in practice, but I'd prefer to set a good example and use `JSON.stringify` instead, in case this idea is reused somewhere where it _does_ matter.

Reply:
So then the strings will not be a string of all the answers, but a super literal string of the array (or arrays for Matcher) made into a strings and then combined to a string? 
<img width="507" alt="image" src="https://github.com/user-attachments/assets/2562f219-60cf-4e65-8d5b-9d2e3f8d8481" />

Reply:
Updated in this PR and the Matcher PR, but let me know if my understanding is incorrect :) Thanks!